communitybsl.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for 1C (BSL)
 homepageUrl=https://1c-syntax.github.io/sonar-bsl-plugin-community/en
-archivedVersions=1.8.1,1.9.0
-publicVersions=1.10.0
+archivedVersions=1.9.0,1.10.0
+publicVersions=1.11.0
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-communitybsl-plugin
 
+1.11.0.description=13 new rules, bug fixes, support new reporters
+1.11.0.sqVersions=[8.9,LATEST]
+1.11.0.date=2022-05-25
+1.11.0.changelogUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/tag/v1.11.0
+1.11.0.downloadUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/download/v1.11.0/sonar-communitybsl-plugin-1.11.0.jar
+
 1.10.0.description=4 new rules, bug fixes.
-1.10.0.sqVersions=[8.9,LATEST]
+1.10.0.sqVersions=[8.9,9.1.*]
```

Comment:
The current version is 9.4, so that's where this range should end. Otherwise your users on 9.2+ won't be offered the upgrade.

Reply:
Thanks a lot, Ann! 
Fixed